src/ol/layer/WebGLPoints.js

```diff
@@ -72,24 +71,24 @@ class WebGLPointsLayer extends Layer {
     super(baseOptions);
 
     /**
-     * @type {import('../style/LiteralStyle.js').LiteralStyle}
+     * @private
+     * @type {import('../webgl/ShaderBuilder.js').StyleParseResult}
      */
-    this.style = options.style;
-
-    assert(this.style.symbol !== undefined, 65);
```

Comment:
@jahow this is not directly related to this pull request but the symbol property is marked as optional [in the typedef](https://github.com/openlayers/openlayers/blob/v6.0.1/src/ol/style/LiteralStyle.js#L9).

would it make sense to make it as mandatory?

Reply:
Sure! it might change when/if we handle other kind of geometries. Thanks